Algorithm/Number_theory/readme.md

```diff
@@ -37,13 +33,12 @@
 - [Discrete Root](https://cp-algorithms.com/algebra/discrete-root.html)
 - [Montgomery Multiplication](https://cp-algorithms.com/algebra/montgomery_multiplication.html)
 
+<hr>
+
+
+# Questions related to Number Theory
 
-## Questions related to Number Theory
-- Counting number of digits in factorial using Kamenetsky's Formula ----> [Java](Code/Java/Kamenetsky_Formula.java)
 - GCD of two number using Euclidean Algorithm ----> [Java](/Code/Java/GCD.java)
-- All Unique Prime Factors ----> [Java](/Code/Java/Uniquefactor.java)
-- Check Prime Number ----> [Java](/Code/Java/check_prime.java)
-- Fibonacci Series ----> [Java](/Code/Java/Fibonacci_Series.java)
-- Determining if a number is a prime number or not using Trial Division Algorithm ---> [C++](/Code/C++/prime_number.cpp)
-- Factorial of a number without recursion ---->[Python](/Code/Python/factorial_without_recursion.py)
-- Check Trimorphic number ---->[C++](/Code/C++/Trimorphic_number.cpp)
```

Comment:
don't make any othet changes on this readme file.... don't delete already existing text... just add your part at last